contributors/guide/scalability-good-practices.md

```diff
@@ -102,68 +96,62 @@ func (c *ControllerX) Run() {
 }
 ```
 
-This looks fine-ish if you don't know that LIST are very expensive calls. Object can have size of few kilobytes, and we can have 150k of those. This means that LIST would need to send those hundreds of megabytes through the network, not to mention the fact that API server would need to do few conversions along the way of all this data. It's not the end of the world, but it needs to be minimized. Especially that the solution is simple (quoting Clayton):
+This may look OK... but only if you do not know that LIST are very expensive calls. Objects can have sizes of a few kilobytes, and there can be 150,000 of those. This means LIST would need to send hundreds of megabytes through the network, not to mention the API server would need to do conversions of all this data along the way. It is not the end of the world, but it needs to be minimized. The solution is simple (quoting Clayton):
 
->As a rule, use informer. If using informer, use shared informers. If your use case doesn't look like an informer, look harder. If at the very end of that, and it still doesn't look like an informer, maybe use something else after talking to someone. But probably use informer.
+>As a rule, use informer. If using informer, use shared informers. If your use case does not look like an informer, look harder. If at the very end of that it still does not look like an informer, consider using something else after talking to someone. But probably use informer.
 
-`Informer` is our library that provides a read interface of the store - it's a read-only cache that provides you a local copy of the store that will contain only object that you're interested in (matching given selector). From it you can GET, LIST, or do whatever read operations you want. `Informer` also allows you to register functions that will be called when an object is created, modified or deleted, which is what most people want.
+`Informer` is our library that provides a read interface to the store - it is a read-only cache that provides you with a local copy of the store that contains only the object in which you are interested (matching given selector). From it you can GET, LIST, or do whatever read operations you desire. `Informer` also allows you to register functions that will be called when an object is created, modified or deleted, which is what most people want.
```

Comment:
"only the object you are interested in" - be bold, and end that sentence with a preposition. I promise, only Word believes it is bad.
Delete "which is what most people want."

Reply:
It isn't just Word, it was 6+ ft tall Sister Joan of the Sisters of Notre Dame. She tattoo'd that on my brain. But since I have a record that you asked for it, you got it. :)